<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessingQueue</class>
 <widget class="QWidget" name="ProcessingQueue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProcessingQueue</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="vLayoutProcessingQueue">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>操作队列</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="9,1,1">
     <item>
      <widget class="QLabel" name="labelPreviewImg">
       <property name="text">
        <string>大图在这里</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPreviewInfo">
       <property name="text">
        <string>信息在这里</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBack">
       <property name="text">
        <string>回退</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
